### PR TITLE
Features/#293 power2gas

### PR DESCRIPTION
**Pull request by AmeliaNadal**
Unfortunatly, I haven't tested it in normal mode.

### CODE REVIEW

**Comment by ClaraBuettner** [comment]
This branch is ~190 commits behind dev. @AmeliaNadal could you merge the recent dev branch into your branch? I would like to test this branch after the merge to not run into problems that are already solved on dev.

**Comment by AmeliaNadal** [comment]
Thanks for the advise @ClaraBuettner, I've tried to merge the current dev into this branch. 
It seems to run without problem in test mode. Unfortunatly, I still haven't tested it in normal mode.

File: src/egon/data/processing/power2gas/__init__.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    new_id = next_id('link')
+        
+    gdf_AC = gpd.read_postgis(sql_AC, engine, crs=4326)
+    gdf_gas = gpd.read_postgis(sql_gas, engine, crs=4326)
+
+    
+    if gdf_AC.size == 0:
+        print("WARNING: No data returned in grid.egon_pf_hv_bus WHERE carrier = 'AC'")
+    elif gdf_gas.size == 0: 
+        print("WARNING: No data returned in grid.egon_pf_hv_bus WHERE carrier = 'gas'")
```
Comment (ClaraBuettner) [inline src/egon/data/processing/power2gas/__init__.py:39]: The function [db.select_geodataframe()](https://github.com/openego/eGon-data/blob/255d9653ab8b24c4a3c63fb9c06d97ef3a2262c8/src/egon/data/db.py#L164-L192) includes this check. If you use this, you don't have to do it manually.

File: src/egon/data/cli.py [hunk 2]
```diff
@@ -141,6 +141,7 @@
     help=("Specify the port on which airflow runs."),
     show_default=True,
 )
+
```
Comment (ClaraBuettner) [inline src/egon/data/cli.py:144]: Do you wanted to change something in this file? I'm just wondering why this file changed on your branch.

**Comment by ClaraBuettner** [comment]
I run the branch successfully in test mode. 
I couldn't really understand what you are doing in `insert_power2gas()` because there is no documentation. Could you add a docstring to the method which describes briefly what you are doing or add some comments to the code?

File: src/egon/data/cli.py (hunk 2 above)
Comment (AmeliaNadal) [reply src/egon/data/cli.py:144]: No, I didn't want to change anything there. This added line is a typo, I'm sorry. Should I remove it?

File: src/egon/data/processing/power2gas/__init__.py (hunk 1 above)
Comment (AmeliaNadal) [reply src/egon/data/processing/power2gas/__init__.py]: Thank you for the advise. I've changed it.

**Comment by ClaraBuettner** [comment]
The renaming of the `grid.egon_pf_hv_` tables to `grid.egon_etrago_` was merged to dev. Could you also adjust this in this branch?

**Comment by ClaraBuettner** [comment]
I run the branch again and noticed an error in a task which is not related to power2gas. (The renaming of etrago input tables was missing in some of my heat scripts.) I fixed this in one commit on this branch. I hope this is okay for you, otherwise I can also undo it.
All tasks run successfully now, so I think you can merge this.

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**